docs/concepts/cluster-administration/guaranteed-scheduling-critical-addon-pods.md

```diff
@@ -48,10 +48,8 @@ It's enabled by default. It can be disabled:
 
 ### Marking add-on as critical
 
-To be critical an add-on has to run in `kube-system` namespace (configurable via flag)
-and have the following annotations specified:
-
-* `scheduler.alpha.kubernetes.io/critical-pod` set to empty string
-* `scheduler.alpha.kubernetes.io/tolerations` set to `[{"key":"CriticalAddonsOnly", "operator":"Exists"}]`
```

Comment:
I think the current wording is a bit clearer in distinguishing annotation vs. field. They are very similar but I think for someone who is not familiar with the concepts, the version I wrote is more explicit and thus less likely to cause confusion.